backend/composer/api/serializers.py

```diff
@@ -683,6 +683,11 @@ class Meta:
 
 class KnowledgeStatementSerializer(ConnectivityStatementSerializer):
     """Knowledge Statement"""
+    def to_representation(self, instance):
```

Comment:
suggestion: @D-GopalKrishna can't we add ```
exclude = ('forward_connection', )
``` to the inner Meta class?

Reply:
thanks @zsinnema! made change as @ddelpiano mentioned.